docs/wdc_oauth_tutorial.md

```diff
@@ -897,544 +960,598 @@ Web Data Connector `.js` library provides a default `init` function, so
-       var dataToReturn = [];
-       var hasMoreData = false;
-
-       var accessToken = tableau.password;
-       var connectionUri = getVenueLikesURI(accessToken);
-
-       var xhr = $.ajax({
-           url: connectionUri,
-           dataType: 'json',
-           success: function (data) {
-               if (data.response) {
-                   var venues = data.response.venues.items;
-                   var ii;
-                   for (ii = 0; ii < venues.length; ++ii) {
-                       var venue = {'Name': venues[ii].name,
-                                    'Latitude': venues[ii].location.lat,
-                                    'Longitude': venues[ii].location.lng,
-                                    'Checkin Count': venues[ii].stats.checkinsCount};
-                       dataToReturn.push(venue);
-                   }
-
-                   tableau.dataCallback(dataToReturn, lastRecordToken, hasMoreData);
-               }
-               else {
-                   tableau.abortWithError("No results found");
-               }
-           },
-           error: function (xhr, ajaxOptions, thrownError) {
-               // If the connection fails, log the error and return an empty set.
-               tableau.log("Connection error: " + xhr.responseText + "\n" +
-                            thrownError);
-               tableau.abortWithError("Error while trying to connect to Foursquare.");
-           }
-       });
-   };
-
-   // Register the tableau connector--call this last
-   tableau.registerConnector(myConnector);
-
-   })();
-```
 
-Next
-----
+---
 
-In the next part of the tutorial, you'll test the web data connector
-using the simulator. You'll also learn why you cannot use the simulator
-alone when your connector uses OAuth.
 
-[Web Data Connector OAuth Tutorial Part 8: Test the Connector in the
-Simulator](wdc_tutorial_oauth_client_test_in_simulator.html)
+# Step 9 Test the Connector in the Simulator
 
+[(Back to top)](#top)
 
-What you'll do
---------------
 
 In this part of the tutorial, you'll test the finished connector in the
 simulator that's part of the Web Data Connector SDK.
 
-Test the finished connector in the simulator
---------------------------------------------
+## Test the finished connector in the simulator
+
 
-You can now test the web data connector in the simulator.
+You can now test the web data connector in the simulator. If you haven't done so already, you need to follow the steps in [Getting Started]({{ site.baseurl }}/docs/index.html) to install the components needed to run the simulator.
 
-1.  Start the simulator by entering the following in your browser:
+1.  Open a Command Prompt or shell window and navigate to the `webdataconnector` folder. If you haven't done so already, start the web server that runs the simulator.
+
+     `npm start`
+
+2.  Start the simulator by entering the following in your browser:
 
     `localhost:88888/simulator/`
```

Comment:
typo I guess. should be localhost:8888

Reply:
@ashwinar I'm not sure what you mean about the typo.  To run the simulator, I have to type the full URL  `localhost:8888/simulator`   If I type localhost:8888 in the top-level folder, I see the Help pages.